designs/administrator/4.3/quay-image-security/index.md

```diff
@@ -0,0 +1,19 @@
+---
+parent: Administrator
+version: 4.3
+---
+
+# Quay Image Security Integration
+
+Users can now install the Container Security Operator to bring Quay and Clair metadata into OpenShift. Installing this operator enables cluster administrators to monitor known container image vulnerabilities in pods running on their Kubernetes cluster. After deployment, there is no additional configuration for the operator to start querying the container registry for vulnerability data. If the registry supports image scanning, like Quay, then the vulnerabilities found will be exposed to the user by populating the **ImageManifestVuln resource** list and surfacing it on the cluster dashboard.
+
+## Vulnerability details on the cluster dashboard
+
+![Cluster dashboard security card 1](img/image1-73.png)
+- Once the Container Security Operator has been installed, the **Quay Image Security** status is shown on the cluster dashboard Status card.
+- The card will list the number of vulnerabilities found and will provide a link to access more detailed information on the type of vulnerabilities.
+- Clicking on the link opens a popover with the breakdown of vulnerability by severity. Critical and high vulnerabilities will be listed first, followed by medium and lower risk. Fixable vulnerabilities are listed with two links.
+- One link is the vulnerability name that goes out to the vulnerability view for the Quay instance the image is hosted on (for example, Quay.io), and the other is a link to view the Image Manifest Vulnerability resource details in the affected namespace.
+
+![Cluster dashboard security card 5](img/image3-55.png)
+- Only the top five most severe vulnerabilities will be listed in the popover. For cases where there are more than five, users can navigate to the custom resource definition list to view all Image Manifest Vulnerability instances under the **ImageManifestVuln** resource.
```

Comment:
Minor nit, but if there are more than five vulnerabilities, will there be some sort of **View all** or **See more** link or will users have to navigate on their own to the full list of instances?

Reply:
+1, my first thought would be "View all" below the bottom-most vulnerability name like other similar popovers, although those names are about to become external links so an internal one below them might be a little strange... Maybe directly above the top-most "1 namespace" when there are more than 5? Any preferences?

If we update this image with a functional change, maybe a small note saying this wasn't in the 4.3 implementation would be warranted? Has this popover been tweaked in 4.4? 🤷‍♂️

Reply:
agree!  Having a View all link sounds consistent with what we're doing in other areas

Reply:
Thanks all! I agree this is needed and both the developer and I were surprised we missed it. I am working with him to get it added via a bug now, not sure what release it will get incorporated in. There were no changes to this popover otherwise in 4.4.

We are working out the exact location and terminology of the link, perhaps **View All Vulnerabilities** since that list is otherwise only showing the fixable vulnerabilities and the list they are taken to would have both fixable and not-fixable? We are also then just considering always having the link present (as long as there is at least 1 fixable or not fixable vulnerability) as a way for the user to access the complete list?

I'll keep the PR updated with what we decide.

Reply:
Thanks Peter and all for the comments! @itsptk Let us know when this is ready to go once you've made the updates and we can merge